proto/directions.proto

```diff
@@ -13,61 +13,6 @@ message DirectionsLeg {
     optional bool has_time_restrictions = 4; // Does the route contain any time restrictions?
   }
 
-  message TransitInfo {
```

Comment:
these things all existed also in the trip proto, so i moved them all to tripcommon. this changes the namespace they are located within of course so the rest of the code changes were all about that